src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Replace the hardcoded value of '7' with a constant that explains what this value represents, such as 'LONG_TERM_RENTAL_DAYS'. Similarly, replace '50' with a constant like 'LONG_TERM_DISCOUNT'.